lib/factory.js

```diff
@@ -1,6 +1,7 @@
 /*!
  * Copyright (c) 2022-2023 Digital Bazaar, Inc. All rights reserved.
  */
+import arrayBufferToBuffer from 'arraybuffer-to-buffer';
```

Comment:
This should be removed -- we don't need another library here or extra conversions to APIs that aren't available on the Web and likely never will be. There are a lot of examples of wrapping `ArrayBuffers` in `Uint8Arrays`, which are constrained views on `ArrayBuffers` (and designed for that very purpose). If `webcrypto.subtle.sign` is returning an `ArrayBuffer`, it should be trivial to wrap it in a `Uint8Array`.

Reply:
I was also hesitant to include this here and wanted your thoughts. My reasoning for converting to `Buffer` can be found in my comment above.

Reply:
Refer to latest comment on thread above. Fix incoming...